superset/sqllab/api.py

```diff
@@ -121,9 +121,7 @@ def export_csv(self, client_id: str) -> CsvResponse:
         """
         result = SqlResultExportCommand(client_id=client_id).run()
 
-        query = result.get("query")
-        data = result.get("data")
-        row_count = result.get("count")
+        query, data, row_count = result["query"], result["data"], result["count"]
```

Comment:
Oh how I wish we had JS-like destructuring in Python..
```
        { query, data, count: row_count } = result
```

Reply:
it's cool for sure